src/features/edge-layout/model.ts

```diff
@@ -28,11 +28,15 @@ export interface EdgeLayoutable extends SModelExtension {
 export function isEdgeLayoutable<T extends SModelElement>(element: T): element is T & SChildElement & BoundsAware & EdgeLayoutable {
     return element instanceof SChildElement
         && element.parent instanceof SRoutableElement
-        && 'edgePlacement' in element
+        && checkEdgeLayoutable(element)
         && isBoundsAware(element)
         && element.hasFeature(edgeLayoutFeature);
 }
 
+function checkEdgeLayoutable(element:SChildElement): element is SChildElement& EdgeLayoutable{
```

Comment:
The linter creates warnings because of inconsistent whitespace. Please run `yarn lint` and correct the warnings.

Reply:
fixed inconsistent whitespace